docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -25,7 +27,7 @@ The ``<participant>`` element has two attributes defined: ``profile_name`` and `
      - Use
    * - ``profile_name``
      - Sets the name under which the ``<participant>`` profile is registered in the DDS Domain, |br|
-       so that it can be loaded later by the |DomainParticipantFactory|, as shown in |br|
+       so that it can be loaded later by the |DomainParticipantFactory-api|, as shown in |br|
```

Comment:
[DomainParticipantFactory] links to DomainParticipant, should link to the Factory